src/main/webapp/join-session.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Join Session</title>
+    <link href="styles/join-style.css" rel="stylesheet" />  
+    <script src="scripts/join-script.js"></script>
```

Comment:
Is this something we're still using? You get redirected to the insession page, right? Which Chris already wrote

Reply:
i believe so, a page to ask for the name and session-id to check against the records is still necessary.

Reply:
I don't see a join-script.js file in the src